packages/core/ios/EmbraceManager.swift

```diff
@@ -598,8 +599,6 @@ class EmbraceManager: NSObject {
 
         if !parentSpanId.isEmpty, let parent = spanRepository.get(spanId: parentSpanId) {
             spanBuilder?.setParent(parent)
-        } else {
-            spanBuilder?.markAsKeySpan()
```

Comment:
note: deprecated, `emb.key` is no longer valid and it's not part of the span attributes

Reply:
@ArielDemarco @NachoEmbrace is this cleanup correct? or is there anything else I need to implement to cover this scenario? (given the fact this method is now deprecated). I would like this confirmed before I release this upgrade (planed for today/tomorrow)

Reply:
Confirmed by @NachoEmbrace , no action item required. This can be removed completely.

Reply:
looks like a couple spots remaining to cleanup on our end:
* packages/core/ios/EmbraceManager.swift:760
* integration-tests/helpers/span.ts:128
* integration-tests/helpers/span.ts:151

Reply:
done